tools/browserutils/browser.py

```diff
@@ -0,0 +1,268 @@
+import logging
+import os
+import re
+import stat
+from abc import ABCMeta, abstractmethod
+from configparser import RawConfigParser
```

Comment:
configparser is 3.x while ConfigParser is 2.x noting that the [web-platform-tests documentation](https://github.com/w3c/web-platform-tests#running-the-tests) states that "test environment requires Python 2.7+".